app/src/main/java/com/hamdan/forzenbook/theme/Theme.kt

```diff
@@ -98,7 +100,6 @@ fun ForzenBookTheme(content: @Composable () -> Unit) {
             )
         )
     }
-    Log.v("Hamdan", "Color:${replacementColors.colors.secondaryVariant.toArgb()}")
```

Comment:
Needed?

Reply:
no it wasn't, I just had forgotten to remove this when I was testing Colors with theme a while ago and it just sat there for a while not getting removed.